ava.config.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I wish this file could live in the task directory instead of the root folder (we're trying to clean it up). From reading https://github.com/avajs/ava/blob/main/docs/06-configuration.md, I suspect this isn't possible, but can we make sure?

Alternatively, is there a way to pass in the  ignored paths via the CLI?

Reply:
No way to define by CLI or elsewhere, unfortunately. 

But we don't need to tie this to ava. In fact, it might be best as a standalone task:

```
amp make-extension-e2e
```

I've made that change instead, but I wonder how it's best to set up its targets. Ideally, it should run when `make-extension` changes, but also when the runtime source changes so that the template stays up-to-date.

What do you think?

Reply:
You can run any `amp` task during CI when any target in a list of targets changes. So this can be triggered by the `RUNTIME` target in addition to whatever target you plan on adding for the task itself. For a full example, see how `amp validator` is triggered in `validator-tests.js`.

Reply:
Cool. Let me know if the `checks.js` setup makes sense to you.